linker/Tests/Mono.Linker.Tests.Cases.Expectations/Assertions/BaseInAssemblyAttribute.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace Mono.Linker.Tests.Cases.Expectations.Assertions
+{
+	public abstract class BaseInAssemblyAttribute : BaseExpectedLinkedBehaviorAttribute {
```

Comment:
What's difference between BaseInAssemblyAttribute and BaseExpectedLinkedBehaviorAttribute ?